rpcs3/Emu/Cell/lv2/lv2.cpp

```diff
@@ -459,7 +459,7 @@ const std::array<std::pair<ppu_intrp_func_t, std::string_view>, 1024> g_ppu_sysc
 	NULL_FUNC(sys_sm_get_tzpb),                             //408 (0x198)  PM
 	NULL_FUNC(sys_sm_get_fan_policy),                       //409 (0x199)  PM
 	BIND_SYSC(_sys_game_board_storage_read),                //410 (0x19A)
-	NULL_FUNC(sys_game_board_storage_write),                //411 (0x19B)
+	BIND_SYSC(_sys_game_board_storage_write),               //411 (0x19B)
```

Comment:
Why adding random underscores?

Reply:
> Why adding random underscores?

It was your suggestion 🙂 https://github.com/RPCS3/rpcs3/pull/12597#discussion_r962124379

Reply:
But there there was an actual collision with an existing function, here it just makes no sense.

Reply:
There's already sys_game_board_storage_write() in rpcs3/Emu/Cell/Modules/sys_game_.cpp.